src/components/ProfilePage/Rewards/RewardsCarousel.tsx

```diff
@@ -8,171 +8,53 @@ import {
-            <FlexCenterRowBox>
-              <Box>
-                <EarnedTypography
-                  color={theme.palette.mode === 'dark' ? '#ffffff' : '#000000'}
-                >
-                  You've earned:
-                </EarnedTypography>
-              </Box>
-              <RewardsAmountBox
-                isSuccess={isMerklSuccess}
-                rewardAmount={rewardAmount}
-                isConfirmed={isConfirmed}
-              />
-            </FlexCenterRowBox>
-            <ClaimButtonBox>
-              <Button
-                disabled={
-                  isPending ||
-                  isConfirming ||
-                  isConfirmed ||
-                  (!!rewardAmount && rewardAmount === 0) ||
-                  (isMerklSuccess && !rewardAmount)
-                }
-                variant="primary"
-                aria-label="Claim rewards"
-                size="large"
-                styles={{
-                  opacity:
-                    isPending ||
-                    isConfirming ||
-                    isConfirmed ||
-                    (!!rewardAmount && rewardAmount === 0) ||
-                    (isMerklSuccess && !rewardAmount)
-                      ? 0.3
-                      : undefined,
-                  alignItems: 'center',
-                  padding: '16px',
-                  width: '100%',
-                }}
-                onClick={() => handleClick()}
+          <FlexCenterRowBox>
+            <Box>
+              <EarnedTypography
+                color={theme.palette.mode === 'dark' ? '#ffffff' : '#000000'}
               >
-                <Typography
-                  fontSize="16px"
-                  lineHeight="18px"
-                  fontWeight={600}
-                  color={theme.palette.white.main}
-                >
-                  {isPending || isConfirming
-                    ? 'CLAIMING...'
-                    : isConfirmed
-                      ? 'CLAIMED'
-                      : 'CLAIM REWARDS'}
-                </Typography>
-              </Button>
-            </ClaimButtonBox>
-            {hash ? (
-              <a
-                href={`${PROFILE_CAMPAIGN_SCANNER}/tx/${hash}`}
-                target="_blank"
-                style={{
-                  textDecoration: 'none',
-                  color: 'inherit',
-                  marginLeft: '32px',
-                }}
-                rel="noreferrer"
-              >
-                <RewardsOpenIconButton>
-                  <OpenInNewIcon
-                    sx={{ height: '32px', color: theme.palette.white.main }}
+                Rewards Earned
+              </EarnedTypography>
+            </Box>
+          </FlexCenterRowBox>
+          {availableRewards.map((availableReward, i) => {
+            const amount = availableReward.amountToClaim;
+            return (
+              <Box key={i + availableReward.address}>
+                {amount > 0 && isMerklSuccess ? (
```

Comment:
Lately, in the reviews of Sebastien he recommended to only use && to replace the ternary operator which returns null. But this works and we still have that pattern applied a lot of times!